synapse/models/person.py

```diff
@@ -45,9 +45,16 @@ def getModelDefs(self):
                 )),
                 ('ps:person', {}, (
                     ('dob', ('time', {}), {
-                        'doc': 'The Date of Birth (DOB) if known.',
+                        'doc': 'The date on which the person was born.',
+                    }),
+                    ('dod', ('time', {}), {
```

Comment:
ps:persona is missing this dod property

Reply:
Purposely left this out due to an up-coming discussion where we may deprecate the entire `ps:persona`